pkg/container-runtime/dockerInstaller.go

```diff
@@ -0,0 +1,70 @@
+// Copyright © 2022 Alibaba Group Holding Ltd.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package container_runtime
+
+import (
+	"fmt"
+	"net"
+
+	"github.com/sealerio/sealer/pkg/infradriver"
+)
+
+type ContainerdInstaller struct {
```

Comment:
If this file is named by dockerInstaller.go, then ContainerdInstaller should be located in another file which is named by containerdInstaller.go.

In addition, the file structure should be :
pkg/container-runtime/installer/docker.go 
pkg/container-runtime/installer/containerd.go